### PR TITLE
Update pt-BR translation

### PR DESCRIPTION
**Pull request by samuelrafo**

### CODE REVIEW

File: Files/MultilingualResources/Files.pt-BR.xlf [hunk 1]
```diff
@@ -3563,37 +3563,37 @@ Usamos o App Center para controlar o uso do aplicativo, encontrar bugs e corrigi
...
         </trans-unit>
         <trans-unit id="OpenLogLocation.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>Open log location</source>
-          <target state="new">Open log location</target>
+          <target state="translated">Abrir local de log</target>
         </trans-unit>
         <trans-unit id="SettingsAboutOpenLogLocationButton.ToolTipService.ToolTip" translate="yes" xml:space="preserve">
           <source>Open log location</source>
-          <target state="new">Open log location</target>
+          <target state="translated">Abrir local de logn</target>
```
Comment (BanCrash) [inline Files/MultilingualResources/Files.pt-BR.xlf:3590]: Shouldn't be this also "log" instead "logn"? I don't speak portuguese but on the other strings you let them as "log".

Comment (samuelrafo) [reply Files/MultilingualResources/Files.pt-BR.xlf:3590]: Ooh its true, i missed a character, thanks for warning.

Comment (BanCrash) [reply Files/MultilingualResources/Files.pt-BR.xlf]: > Ooh its true, i missed a character, thanks for warning.

No problem!

**Review by BanCrash — APPROVED** [review]
LGTM

**Pull request merged**